src/content/platform-overview/cluster-management/cluster-api/architecture/index.md

```diff
@@ -87,15 +87,15 @@ The Kubernetes cluster is represented in VMware Cloud Directior by a vAPP of the
 In order to offer persistent storage that is decoupled from the virtual machines, the container storage interface creates a Named Disk that can be attached or detached from the VM according to whether or not the persistent volume claim (PVC) is bound to a pod or not. Named disks currently only support Read-Write-Only (RWO) with block storage backed named disks.
 
 {{< /tab >}}
-{{< tab id="flags-aws" title="AWS (CAPA)">}}
+{{< tab id="cluster-capa_ec2" for-impl="capa_ec2" >}}
```

Comment:
I assumed it was EC2 and not EKS here. Can someone confirm?

Reply:
The description mostly matches CAPA EC2, and our EKS documentation will anyway follow in the next weeks.